experimental/Experimental.jl

```diff
@@ -1,7 +1,12 @@
+import Glob
+for fname in Glob.glob("*/src/main.jl", "experimental")
+  fname = SubString(fname, 14, length(fname))
```

Comment:
I dislike magic constants like the 14 here. Where does it come from? `length("experimental/") == 13`... 

Anyway, isn't this code executed within `experimental anyway? That would suggest
```suggestion
for fname in Glob.glob("*/src/main.jl")
```